### PR TITLE
8306135: Clean up and open source some AWT tests

### PR DESCRIPTION
**Pull request by azuev-java**
Cleaning up and open sourcing three AWT tests

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8306135](https://bugs.openjdk.org/browse/JDK-8306135): Clean up and open source some AWT tests

### Reviewers
 * [Alexander Zvegintsev](https://openjdk.org/census#azvegint) (@azvegint - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13518/head:pull/13518` \
`$ git checkout pull/13518`

Update a local copy of the PR: \
`$ git checkout pull/13518` \
`$ git pull https://git.openjdk.org/jdk.git pull/13518/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13518`

View PR using the GUI difftool: \
`$ git pr show -t 13518`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13518.diff">https://git.openjdk.org/jdk/pull/13518.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13518#issuecomment-1513661787)

### CODE REVIEW

File: test/jdk/java/awt/Cursor/SingleColorCursorTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import java.awt.Button;
+import java.awt.Cursor;
+import java.awt.EventQueue;
+import java.awt.Frame;
+import java.awt.Panel;
+import java.awt.Point;
+import java.awt.Toolkit;
+import java.awt.image.BufferedImage;
+import java.lang.reflect.InvocationTargetException;
+public class SingleColorCursorTest extends Panel {
```
Comment (lawrence-andrew) [inline test/jdk/java/awt/Cursor/SingleColorCursorTest.java:42]: Can you add empty line between class and the import statement

**Comment by lawrence-andrew** [comment]
All the 3 tests are new to open repo and there copy right years are different.

File: test/jdk/java/awt/Dialog/DialogAsParentOfFileDialog.java [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+            errors.append(nl);
+            errors.append("DialogAsParentOfFileDialog FAILED");
+        }
+
+        if (!errors.isEmpty()) {
+            throw new RuntimeException("Following tests failed:" + errors);
+        }
+    }
+
+    public static void main(String[] args) throws InterruptedException, InvocationTargetException {
```
Comment (lawrence-andrew) [inline test/jdk/java/awt/Dialog/DialogAsParentOfFileDialog.java:182]: Can you set to 80-column limit.

File: test/jdk/java/awt/Cursor/SingleColorCursorTest.java (hunk 1 above)
Comment (azuev-java) [reply test/jdk/java/awt/Cursor/SingleColorCursorTest.java:43]: Done

File: test/jdk/java/awt/Dialog/ComponentShownEvent.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                }
+                if (f != null) {
+                    f.setVisible(false);
+                    f.dispose();
+                }
+            });
+        }
+    }
+
+    public static void main(String[] args) throws InterruptedException, InvocationTargetException, AWTException {
```
Comment (lawrence-andrew) [inline test/jdk/java/awt/Dialog/ComponentShownEvent.java:86]: Can you set to 80-column limit.

**Comment by lawrence-andrew** [comment]
Set to 80-column limit where ever it required.

File: test/jdk/java/awt/Dialog/ComponentShownEvent.java (hunk 3 above)
Comment (azuev-java) [reply test/jdk/java/awt/Dialog/ComponentShownEvent.java:86]: Now trying to keep the lines lengths reasonable.

File: test/jdk/java/awt/Dialog/DialogAsParentOfFileDialog.java (hunk 2 above)
Comment (azuev-java) [reply test/jdk/java/awt/Dialog/DialogAsParentOfFileDialog.java]: Done

**Review by azvegint — APPROVED** [review]

**Pull request closed without merging**